### ISSUE TITLE
Test failure (Perl 5.34.0)

### ISSUE THREAD

**Issue by robrwo** [briandfoy/cpan-audit#35]
```
t/cli.t ........ ok
t/excludes.t ... ok
t/installed.t .. ok

    #   Failed test at t/json.t line 24.
    #                   ''
    #     doesn't match '(?^:Discovered 1 dependencies)'

    #   Failed test at t/json.t line 25.
    #          got: 'Discovered 1 dependencies
    # '
    #     expected: ''

    #   Failed test at t/json.t line 26.
    #          got: 'HASH(0x3a5d580)'
    #     expected: '1'

    #   Failed test at t/json.t line 29.
    #                   ''
    #     doesn't match '(?^:CPANSATest)'
    # Looks like you failed 4 tests of 4.

#   Failed test 'json file'
#   at t/json.t line 32.
# Looks like you failed 1 test of 1.
t/json.t ....... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
t/query.t ...... ok
t/version.t .... ok

Test Summary Report
-------------------
t/json.t     (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
Files=6, Tests=10,  1 wallclock secs ( 0.02 usr  0.01 sys +  0.67 cusr  0.07 csys =  0.77 CPU)
Result: FAIL
Failed 1/6 test programs. 1/10 subtests failed.
Makefile:928: recipe for target 'test_dynamic' failed
make: *** [test_dynamic] Error 255

```

**Comment by briandfoy** [comment]
Which versions of the module and database is this for?

**Comment by robrwo** [comment]
> Which versions of the module and database is this for?

Latest version from CPAN 20230104.001

**Comment by briandfoy** [comment]
I can't reproduce this. Maybe something in my setup is masking it, but all the CI tests pass this test too. Can you look into what's going wrong using your setup?

**Comment by briandfoy** [comment]
Okay, I can reproduce this now. the t/json.t test wasn't listed in test_manifest, so it wasn't showing up in systems using Test::Manifest. This effectively hid the test.

**Comment by briandfoy** [comment]
I think I have a fix for this and I am sending it to CPAN Testers.

What went wrong:

* I didn't properly update the t/json.t test after I reworked Reneé's pull request to add that feature
* Then, I thought I was running the t/json.t but I wasn't. This hid the error.
* I took a bit of a shortcut to upload the file to PAUSE after #32, which bypasses all my checks that find this sort of stuff.

**Comment by briandfoy** [comment]
You can play with the [GitHub release](https://github.com/briandfoy/cpan-audit/releases/tag/release-20230125.001_002) while I wait for CPAN Testers feedback.

**Comment by briandfoy** [comment]
This fix is in 20230125.002
